### PR TITLE
Add documentation about updating dependencies in batch

### PR DESCRIPTION
**Pull request by Castavo**

### CODE REVIEW

File: content/docs/guides/contribute/batch-updating-dependencies.en.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Here is the process to update dependencies:
+
+1. Change the versions.
+    * *If you're using VSCode* you can install the `crates` extension and run the "update all dependencies" command.  
+    Doing so will update all dependencies to their latest version, and overwrite voluntarily loose version constraints.  
+    Make sure that the new version chosen is stable, and that loose constraints are not overwritten in your commit.
+    * *If you're not*, you can go check the versions used by dependabot in [its PRs](https://github.com/osrd-project/osrd/lls?q=is%3Aopen+label%3Aarea%3Aeditoast+label%3Adependencies) and update the versions manually.
+2. Run `cargo update` to update the Cargo.lock file (even sub-dependencies).
+3. Check that all [dependabot editoast PRs](https://github.com/osrd-project/osrd/lls?q=is%3Aopen+label%3Aarea%3Aeditoast+label%3Adependencies) are included in your update.
```
Comment (flomonster) [inline content/docs/guides/contribute/batch-updating-dependencies.en.md:21]: I get a 404 with this URL

File: content/docs/guides/contribute/batch-updating-dependencies.en.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+We use dependabot on the project to signal when dependencies are outdated. We do not use dependabot to automatically update dependencies, as we want to merge all updates at once and review the changes.
+
+Here is the process to update dependencies:
+
+1. Change the versions.
+    * *If you're using VSCode* you can install the `crates` extension and run the "update all dependencies" command.  
+    Doing so will update all dependencies to their latest version, and overwrite voluntarily loose version constraints.  
+    Make sure that the new version chosen is stable, and that loose constraints are not overwritten in your commit.
+    * *If you're not*, you can go check the versions used by dependabot in [its PRs](https://github.com/osrd-project/osrd/lls?q=is%3Aopen+label%3Aarea%3Aeditoast+label%3Adependencies) and update the versions manually.
```
Comment (flomonster) [inline content/docs/guides/contribute/batch-updating-dependencies.en.md:19]: I get a 404 with this URL

File: content/docs/guides/contribute/batch-updating-dependencies.en.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+## For editoast
+
+We use dependabot on the project to signal when dependencies are outdated. We do not use dependabot to automatically update dependencies, as we want to merge all updates at once and review the changes.
+
+Here is the process to update dependencies:
+
+1. Change the versions.
+    * *If you're using VSCode* you can install the `crates` extension and run the "update all dependencies" command.
```
Comment (flomonster) [inline content/docs/guides/contribute/batch-updating-dependencies.en.md:16]:
```suggestion
    * *If you're using VSCode* you can install the `serayuzgur.crates` extension and run the "update all dependencies" command.  
```

Comment (flomonster) [reply content/docs/guides/contribute/batch-updating-dependencies.en.md:16]: We could also include a ling to https://marketplace.visualstudio.com/items?itemName=serayuzgur.crates?

**Review by flomonster — CHANGES_REQUESTED** [review]
Thanks for the PR! A few things to fix though.

**Comment by hamz2a** [comment]
LGTM

**Review by flomonster — APPROVED** [review]

**Review by Tguisnet — APPROVED** [review]

**Pull request merged**